src/index.html

```diff
@@ -8,6 +8,50 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <!-- <h1>Stars</h1> -->
+
+    <!-- do nth-childow skasoac klase --active -->
```

Comment:
please remove every comment before pushing commit - unless it's really necessary 😉